src/lib/ProductExporter.ts

```diff
@@ -132,7 +163,7 @@ export class ProductExporter {
                 parentCategoriesIds.add(category['enxCPQ__Parent_Category__r']['enxCPQ__TECH_External_Id__c']);
             }
 
-            Util.writeFile('./temp/categories/' + category['Name'] +'_' +category['enxCPQ__TECH_External_Id__c']+ '.json', category);
+            Util.writeFile('./'+this.dir +'/categories/' + category['Name'] +'_' +category['enxCPQ__TECH_External_Id__c']+ '.json', category);
```

Comment:
a gdyby `this.dir` było w metodzie Util.writeFile to miałbyś tylko jedno miejsce w którym musisz napisać this.dir